scarlet/source.py

```diff
@@ -238,50 +222,45 @@ def init_multicomponent_source(sky_coord, frame, observations, bg_rms, flux_perc
         morphs[k] /= morphs[k].max()
 
     # optimal SEDs given the morphologies, assuming img only has that source
-    seds = get_best_fit_seds(morphs, frame, observations)
+    seds = get_best_fit_seds(morphs, frame, observation)
 
     return seds, morphs
 
 
 class PointSource(Component):
-    """Extended source intialized with a single pixel
-
-    Point sources are initialized with the SED of the center pixel,
-    and the morphology of a single pixel (the center) turned on.
-    While the source can have any `constraints`, the default constraints are
-    symmetry and monotonicity.
-
-    Parameters
-    ----------
-    sky_coord: tuple
-        Center of the source
-    frame: `scarlet.observation.Frame`
-        The frame that the model lives in.
-    observations: list of `~scarlet.observation.Observation`
-        Observations to extract SED from.
-    symmetric: bool
-        Whether or not the object is forced to be symmetric
-    monotonic: bool
-        Whether or not the object is forced to be monotonically decreasing
-    center_step: int
-        Number of steps to skip between centering routines
-    delay_thresh: int
-        Number of steps to skip before turning on thresholding.
-        This is useful for point sources because it allows them to grow
-        slightly before removing pixels with low significance.
-    component_kwargs: dict
-        Keyword arguments to pass to the component initialization.
-    """
-
-    def __init__(self, sky_coord, frame, observations, symmetric=False, monotonic=True,
+    def __init__(self, frame, sky_coord, observation, symmetric=False, monotonic=True,
                  center_step=5, delay_thresh=10, **component_kwargs):
-        try:
-            iter(observations)
-        except TypeError:
-            observations = [observations]
+        """Extended source intialized with a single pixel
+
+        Point sources are initialized with the SED of the center pixel,
+        and the morphology of a single pixel (the center) turned on.
+        While the source can have any `constraints`, the default constraints are
+        symmetry and monotonicity.
```

Comment:
This should be in the docstring for the class. The only thing in the constructor docstring should be the parameters.